app/services/availability_service.py

```
@@ -428,20 +431,24 @@ async def check_availability(self, request: AvailabilityRequest) -> Availability
-                        date_str=request.date,
-                        hour_slots=hour_slots,
-                        available_slots=res.available_slots if is_partial else None,
-                    )
+                    try:
+                        total_price = self.calculate_total_price(
+                            room_detail=room_detail,
+                            date_str=request.date,
+                            hour_slots=hour_slots,
+                            available_slots=res.available_slots if is_partial else None,
+                        )
+                    except Exception as e:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)